### PR TITLE
Business intelligence merchants

### PR DESCRIPTION
**Pull request by meyerhoferc**
Ads simplecov and an updated test for finding customers with pending transactions for a merchant

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -25,6 +25,7 @@ gem 'puma', '~> 3.0'
 # Use Rack CORS for handling Cross-Origin Resource Sharing (CORS), making cross-origin AJAX possible
 # gem 'rack-cors'
 gem 'active_model_serializers', '~> 0.10.0'
+gem 'simplecov', :require => false, :group => :test
 group :development, :test do
```
Comment (scmountain) [inline Gemfile:29]: @meyerhoferc move this to test env?

**Review by scmountain — COMMENTED** [review]
env gem question

File: Gemfile (hunk 1 above)
Comment (meyerhoferc) [reply Gemfile:29]: it's in the test environment because it's group points to test. it would probably cleaner if moved there, though

Comment (scmountain) [reply Gemfile:29]: I am unfamiliar with that notation so I didn't want to mess with it and ruin your work, we can alway refactor when after our function is 100%

**Pull request merged**